packages/@aws-cdk/aws-iam/lib/policy-statement.ts

```diff
@@ -200,15 +200,15 @@ export class PolicyStatement {
   /**
    * Add a condition to the Policy
    */
-  public addCondition(key: string, value: any) {
+  public addCondition(key: string, value: { [key: string]: string | string[] }) {
```

Comment:
Unfortunately we can't retype this this accurately, because we can't support unions, so retype `string | string[]` as `any`.

We should probably have the following type aliases, no?

```ts
export type Condition = Record<string, any>;
export type Conditions = Record<string, Condition>;
```

Then this whole argument could be typed as `Condition`.

Reply:
Sounds good, thanks for the pointer.